Week3/todo-app/api/createTodoList.js

```diff
@@ -0,0 +1,60 @@
+const logger = require("../logger");
+const dbManager = require("../db");
+const user = require("../user");
+const { successResponse, failureResponse } = require("../response");
+const { responseObject } = require("../config");
+
+const createTodoList = (req, res) => {
+  responseObject.operation = "createTodoList";
+  logger.log(`POST: /list, body: ${JSON.stringify(req.body)}`);
+  const { user: userCredentials, todoList } = req.body;
```

Comment:
Maybe it's just my installed versions but user: userCredentials doesn't work for me. I made a quick workaround to test it.